src/lib/index.js

```diff
@@ -31,6 +31,7 @@ lib.parseDate = datesModule.parseDate;
 var searchModule = require('./search');
 lib.findBin = searchModule.findBin;
 lib.sorterAsc = searchModule.sorterAsc;
+lib.sorterDec = searchModule.sorterDes;
```

Comment:
Thanks good catch.

I was thinking of *Descending*. So `sorterDes` is OK? I have no preference.